### PR TITLE
2nd go at adding translations for non-bible text

### PR DESCRIPTION
**Pull request by dcenatiempo**
2nd try at this using a more conventional i18n() setup
issue #13

### CODE REVIEW

File: assets/translations/index.json [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    "traditional_chinese": "Traditional Chinese",
+    "simplified_chinese": "Simplified Chinese",
+    "additional_info": "Aditional Info",
+    "about_us": "About Us",
+    "font_size": "Font Size",
+    "small": "Small",
+    "medium": "Medium",
+    "large": "Large",
+    "regular": "Regular"
+	},
```
Comment (GyozaGuy) [inline assets/translations/index.json]: A bit of funny whitespace in this file.

File: App/lib/hooks/useI18n.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { selectors } from 'store';
+import defaultTranslations from 'translations';
+
+export default function useI18n(args) {
+  const defaultLanguage = useSelector(selectors.language);
+  const language = args?.language || defaultLanguage;
+  const translations = args?.translations || defaultTranslations;
+
+  function i18n(key) {
+    return get(translations, `[${language}][${key}]`, 'xxx');
```
Comment (GyozaGuy) [inline App/lib/hooks/useI18n.js]: It might be good to return the `key` variable as a fallback here instead of `xxx` so we know which key is faulty.

**Review by GyozaGuy — APPROVED** [review]
Looks good! I'm OK with #27 as well if @clintKunz has strong feelings that way.

**Review by clintKunz — APPROVED** [review]
I'm learning from both prs, happy with either one

File: App/lib/hooks/useI18n.js (hunk 2 above)
Comment (dcenatiempo) [reply App/lib/hooks/useI18n.js]: That's a good idea, I don't like xxx, but wasn't sure what to return...

**Comment by dcenatiempo** [comment]
fixes #13

**Pull request merged**